docs/data-analysis.md

```diff
@@ -90,31 +179,27 @@ print(f"Loaded {len(bt_data)} rows of data for {pair} from {data_location}")
 
 ```python
 # Load strategy using values set above
+from freqtrade.resolvers import StrategyResolver
 strategy = StrategyResolver({'strategy': strategy_name,
                             'user_data_dir': user_data_dir,
                             'strategy_path': strategy_location}).strategy
 
 # Generate buy/sell signals using strategy
-df = strategy.analyze_ticker(bt_data, {'pair': pair})
+df = strategy.analyze_ticker(candles, {'pair': pair})
 ```
 
 ### Display the trade details
 
 * Note that using `data.head()` would also work, however most indicators have some "startup" data at the top of the dataframe.
```

Comment:
I would add a comment here suggesting to use `df.tail()` instead of `data.head()`.

Oddly enough, this command is part of the notebook, but doesn't appear in this code section.